apps/sms/js/thread_ui.js

```diff
@@ -1935,20 +1933,17 @@ var ThreadUI = {
     }
   },
 
-  checkInputs: function thui_checkInputs() {
+  updateSelectionStatus: function thui_updateSelectionStatus() {
```

Comment:
just a note (because we won't change this here): this is called each time we add a new message when we're in edit mode, so this consumes a lot of CPU if we enter edit mode while the thread is still loading.